testsuite/src/main/java/io/netty/testsuite/transport/socket/DatagramUnicastTest.java

```diff
@@ -189,7 +189,7 @@ public void channelRead0(ChannelHandlerContext ctx, Object msgs) throws Exceptio
             for (ChannelFuture future: futures) {
                 future.sync();
             }
-            if (!latch.await(10, TimeUnit.SECONDS)) {
+            if (!latch.await(100000, TimeUnit.SECONDS)) {
```

Comment:
@normanmaurer hmm, this wasn't my comment :)

Reply:
Fixed... doh!